pkgs/development/tools/templ/default.nix

```diff
@@ -0,0 +1,55 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "templ";
+  version = "0.2.476";
+
+  subPackages = [ "cmd/templ" ];
+
+  CGO_ENABLED = 0;
+
+  flags = [
+    "-trimpath"
+  ];
```

Comment:
trimpath is added by buildGoModule [by default](https://github.com/NixOS/nixpkgs/blob/a1422a7f8f12b68e307b92b7daa254f412e9ff96/pkgs/build-support/go/module.nix#L154), this can be removed.